scripts/completion/dvc.bash

```diff
@@ -12,30 +12,36 @@ _dvc_commands='init destroy add import checkout run pull push fetch \
               status repro remove move unprotect gc config remote metrics \
               install root lock unlock pipeline commit'
 
-_dvc_global_options="-h --help -q --quiet -V --verbose"
 _dvc_options="-h --help -v --version"
-_dvc_init="--no-scm -f --force"
-_dvc_destroy="-f --force"
+_dvc_global_options="-h --help -q --quiet -V --verbose"
+
 _dvc_add="-R --recursive -f --file --no-commit"
-_dvc_run="--no-exec -f --file -c --cwd -d --deps -o --outs -O --outs-no-cache -M --metrics-no-cache -y --yes --overwrite-dvc-file --ignore-build-cache --remove-outs --no-commit -w --wdir"
-_dvc_pull="--show-checksums -j --jobs -r --remote -a --all-branches -T --all-tags -d --with-deps -f --force -R --recursive"
-_dvc_push="--show-checksums -j --jobs -r --remote -a --all-branches -T --all-tags -d --with-deps -R --recursive"
+#_dvc_cache=?
```

Comment:
Is this PR still WIP, or are these just reminder notes for the future? If the latter, then please an issue for those instead of these comments, since they are easy to forget about.

Reply:
Sure, the issue has been open already #2069 but indeed I was planning to leave the comments in there also. I guess that's kind of ugly. Removed now.